vsphere/resource_vsphere_content_library.go

```diff
@@ -113,13 +113,13 @@ func resourceVSphereContentLibrary() *schema.Resource {
 						Type:     schema.TypeBool,
 						ForceNew: true,
 						Optional: true,
-						Default:  true,
+						Default:  false,
```

Comment:
I will have to brush up on the subtleties of this situation, I also don't have full context on the provider/product yet, but this change could be perceived as a breaking change?

What happens to users who had both this field and `automatic_sync` unset?

Reply:
As I recall, it will default to `on_demand` still. 

Yes, it could be perceived as a breaking change, but the current experience is opposite of the product defaults.

Happy to help in any way - [message me](mailto://johnsonryan@vmware.com), if needed.

Ryan